src/pages/index.js

```diff
@@ -84,13 +141,104 @@ forms.forEach(function (form) {
   const newFormValidator = new FormValidator(settings, form);
   newFormValidator.enableValidation();
 });
-const userInfo = new UserInfo(
-  profileInfo
-  // profileInfo.nameSelector,
-  // profileInfo.aboutMeSelector
-);
-cardsList.renderItems();
+const userInfo = new UserInfo(profileInfo);
 
 profilePopup.setEventListeners();
-cardForm.setEventListeners();
+// cardForm.setEventListeners();
 imagePopup.setEventListeners();
+
+profilePicturePopup.setEventListeners();
+profileImage.addEventListener("mouseover", () => {
+  profileEditIcon.style.visibility = "visible";
+});
+profileImage.addEventListener("mouseout", () => {
+  profileEditIcon.style.visibility = "hidden";
+});
+profileImage.addEventListener("click", () => {
+  profilePicturePopup.openPopup();
+});
+
+const apiInfo = {
+  url: "https://around-api.en.tripleten-services.com/v1/users/me",
+};
+
+// api.getInitialCards().then((res) => {
+//   const initCards = res;
+//   console.log(initCards);
+//   const cardsList = new Section(
+//     {
+//       data: initCards,
+//       renderer: (cardItem) => {
+//         const newCard = createCard(cardItem);
+//         cardsList.addItem(newCard);
+//       },
+//     },
+//     mediaList
+//   );
+//   cardsList.renderItems();
+// });
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then((res) => {
+    const profileInfo = res[0];
+    const initCards = res[1];
+    userInfo.setUserInfo(profileInfo);
+    profileAvatar.src = profileInfo.avatar;
+    const cardsList = new Section(
+      {
+        data: initCards,
+        renderer: (cardItem) => {
+          const newCard = createCard(cardItem);
+          console.log(cardItem);
+          cardsList.addItem(newCard);
+        },
+      },
+      mediaList
+    );
+    function addCard(data, wrapper) {
+      const newCard = createCard(data);
+      cardsList.addItem(newCard);
+    }
+    const cardForm = new PopupWithForm(newCardModal, (data, button) => {
+      const name = newCardTitle.value;
+      const link = newCardURL.value;
```

Comment:
Only method `_getInputValues` collects all the data of the inputs **inside** `PopupWithForm` and passes it to the call of `submitForm`. So you need to use the collected data rather than the inputs.